yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -216,6 +211,7 @@ private[spark] class Client(
     val fs = FileSystem.get(hadoopConf)
     val dst = new Path(fs.getHomeDirectory(), appStagingDir)
     val nns = getNameNodesToAccess(sparkConf) + dst
+    val distributedUris = new HashSet[String]
```

Comment:
The use of this isn't immediately clear.  Mind adding a short comment above describing its purpose?